rebasehelper/plugins/output_tools/json_.py

```diff
@@ -34,14 +34,14 @@ class JSON(BaseOutputTool):
     EXTENSION: str = 'json'
 
     @classmethod
-    def print_summary(cls, path, results):
+    def print_summary(cls, report_path, results):
         """
         Print JSON summary
 
-        :param path: to the report file
-        :param results: dictionary containing info about rebase
+        :param report_path path to the report
+        :param results ResultsStore instance
```

Comment:
If you want to keep this style, you should add the missing semicolons (as this is a [field list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#field-lists)).

Reply:
Hm, I will rewrite the documentation in output tools, there are not that many docstrings and we should slowly work towards replacing all of these old-style docstrings with napoleon.